### DISCUSSION TITLE
Kan vi skrote /profil/tilgjengelighet i portalen?

### DISCUSSION THREAD

**Discussion by onordbo** [fremtind/jokul#2732] [Annet]
Vi har en hel seksjon som [snakker om universell utforming](https://jokul.fremtind.no/universell-utforming), og innholdet som går på tilgjengelighet under profilseksjonen vår er mildt sagt [ganske halvbakt](https://jokul.fremtind.no/profil/tilgjengelighet) 😅

Her er det såpass lite content og egentlig bare en link videre til UU seksjonen vår. Er denne siden egentlig nødvendig? 👇
<img width="1392" alt="image" src="https://user-images.githubusercontent.com/4263132/154227061-1e94e2ff-1898-4349-86b8-8012c6741eff.png">

**Comment by Murstam** [comment]
Har tenkt det samme. Bare å fjerne spør du meg :)

**Comment by piofinn** [comment]
e9

**Comment by onordbo** [comment]
Vipps, så var den vekk! https://github.com/fremtind/jokul/issues/2740
